src/query/shorthand.ts

```diff
@@ -189,25 +189,25 @@ export function encoding(encQ: EncodingQuery,
  * @param include Dict Set listing property types (key) to be included in the shorthand
  * @param replace Dictionary of replace function for values of a particular property type (key)
  */
-export function fieldDef(encQ: EncodingQuery,
+export function fieldDef(fieldQ: FieldQuery,
```

Comment:
Basically, for ValueQuery we just stringify `value`.   This way string values will become quoted strings e.g., `value: "a"` => `'"a"'`.